Riot/Managers/Analytics/Analytics.m

```diff
@@ -31,7 +31,29 @@
 NSString *const kAnalyticsE2eDecryptionFailureAction = @"Decryption failure";
 
 
-@import PiwikTracker;
+@import MatomoTracker;
+
+@interface MatomoTracker (MatomoTrackerMigration)
++ (MatomoTracker *)shared;
+
++ (void)migrateFromFourPointFourSharedInstance;
+@end
+
+@implementation MatomoTracker (MatomoTrackerMigration)
++ (MatomoTracker *)shared
+{
+    NSDictionary *piwikConfig = [[NSUserDefaults standardUserDefaults] objectForKey:@"piwik"];
+    MatomoTracker *matomoTracker = [[MatomoTracker alloc] initWithSiteId:piwikConfig[@"siteId"] baseURL:[NSURL URLWithString:piwikConfig[@"url"]] userAgent:@"iOSMatomoTracker"];
+    return matomoTracker;
+}
+
++ (void)migrateFromFourPointFourSharedInstance
+{
+    if ([[NSUserDefaults standardUserDefaults] boolForKey:@"migratedFromFourPointFourSharedInstance"]) return;
+    
```

Comment:
Missing here is a call to [self copyFromOldSharedInstance]. However, for some reason, that method stays unresolved. It is defined here: https://github.com/matomo-org/matomo-sdk-ios/blob/187465b4bda57e677af36c9808169d795d4510e0/MatomoTracker/MatomoTracker.swift#L452